{"name": "testcase2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 3\n  +2\n  -3", "trace": "@pytest.mark.run(order=2)\n    @allure.title(\"testcase2\")\n    @allure.description(\"I am first test_case2\")\n    @allure.story('story_2')\n    def test_case2():\n        print('I am case2')\n        LoggerForApitest.writeInfoLog(\"I AM CASE2\")\n>       assert 2 == 3\nE       assert 2 == 3\nE         +2\nE         -3\n\npytest_case\\test_fixtures1.py:48: AssertionError"}, "description": "I am first test_case2", "attachments": [{"name": "log", "source": "1dc641cd-2847-417a-9768-fc164d605f55-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "6dd51d04-8676-4192-9889-137c68b6ede3-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "074de2f0-3c31-464e-b367-ce4d8b4fd9bb-attachment.txt", "type": "text/plain"}], "start": 1647999386693, "stop": 1647999386693, "uuid": "581b39de-e2d0-4185-bf4a-d56d12fa4fad", "historyId": "92f9d3b642298a60c88279ecf235fa2d", "testCaseId": "725ce7cbcf31b4144284ab868efa6e15", "fullName": "pytest_case.test_fixtures1#test_case2", "labels": [{"name": "story", "value": "story_2"}, {"name": "tag", "value": "run(order=2)"}, {"name": "parentSuite", "value": "pytest_case"}, {"name": "suite", "value": "test_fixtures1"}, {"name": "host", "value": "no-delecs-g42-test-wx405299"}, {"name": "thread", "value": "13380-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pytest_case.test_fixtures1"}]}